01-node-tutorial/answers/10-fs-sync.js

```diff
@@ -0,0 +1,20 @@
+const { readFileSync, writeFileSync } = require('fs')
+
+// const fs = require('fs')
+// console.log(fs.readFileSync('../content/first.txt', 'utf8'));
+
+const firstFileBinary = readFileSync('../content/first.txt', 'binary')
```

Comment:
It is very cool that you tried this, however, the goal of the assignment was to write to a file 3 times, using the 'a' flag

Reply:
Changes done